src/cable.cr

```diff
@@ -21,6 +21,7 @@ module Cable
       unauthorized:    "unauthorized",
       invalid_request: "invalid_request",
       server_restart:  "server_restart",
+      remote:          "remote",
```

Comment:
This isn't actually used, but then again, I realized most of this isn't used. ActionCable allows for sending a message of *why* a disconnect happens, but we don't seem to have that currently.